monitoring/mock_uss/geoawareness/parsers/ed269/__init__.py

```diff
@@ -0,0 +1,151 @@
+from enum import Enum
+from typing import List, Any, Optional, Dict
+import arrow
+
+from implicitdict import ImplicitDict, StringBasedDateTime
+
+
+class Restriction(str, Enum):
+    PROHIBITED = "PROHIBITED"
+    REQ_AUTHORISATION = "REQ_AUTHORISATION"
+    CONDITIONAL = "CONDITIONAL"
+    NO_RESTRICTION = "NO_RESTRICTION"
+
+
+class Reason(str, Enum):
+    AIR_TRAFFIC = "AIR_TRAFFIC"
+    SENSITIVE = "SENSITIVE"
+    PRIVACY = "PRIVACY"
+    POPULATION = "POPULATION"
+    NATURE = "NATURE"
+    NOISE = "NOISE"
+    FOREIGN_TERRITORY = "FOREIGN_TERRITORY"
+    EMERGENCY = "EMERGENCY"
+    OTHER = "OTHER"
+
+
+class YESNO(str, Enum):
+    YES = "YES"
+    NO = "NO"
+
+
+class Purpose(str, Enum):
+    AUTHORIZATION = "AUTHORIZATION"
+    NOTIFICATION = "NOTIFICATION"
+    INFORMATION = "INFORMATION"
+
+
+class UASZoneAuthority(ImplicitDict):
+    name: Optional[str]  # max length: 200
+    service: Optional[str]  # max length: 200
+    email: Optional[str]
+    contactName: Optional[str]  # max length: 200
+    siteURL: Optional[str]
+    phone: Optional[str]  # max length: 200
+    purpose: Optional[Purpose]
+    intervalBefore: Optional[str]
+
+
+class VerticalReferenceType(str, Enum):
+    AGL = "AGL"
+    AMSL = "AMSL"
+
+
+class HorizontalProjectionType(str, Enum):
+    Circle = "Circle"
+    Polygon = "Polygon"
+
+
+class CircleOrPolygonType(ImplicitDict):
+    type: HorizontalProjectionType
+    center: Optional[List[float]]  # 2 items
+    radius: Optional[float]  # > 0
+    coordinates: Optional[List[List[float]]]  # min 4 items  # 2 items
+
+
+class UomDimensions(str, Enum):
+    M = "M"
+    FT = "FT"
+
+
+class UASZoneAirspaceVolume(ImplicitDict):
+    uomDimensions: UomDimensions
+    lowerLimit: Optional[int]
+    lowerVerticalReference: VerticalReferenceType
+    upperLimit: Optional[int]
+    upperVerticalReference: VerticalReferenceType
+    horizontalProjection: CircleOrPolygonType
+
+
+class WeekDateType(str, Enum):
+    MON = "MON"
+    TUE = "TUE"
+    WED = "WED"
+    THU = "THU"
+    FRI = "FRI"
+    SAT = "SAT"
+    SUN = "SUN"
+    ANY = "ANY"
+
+
+class ED269TimeType(str):
+    """String that allows values which describe a time in ED-269 flavour of ISO 86001 format.
+
+    ED-269 standard specifies that a time instant type should be in the form of hh:mmS where S is
+    the timezone. However, examples are using the following format: 00:00:00.00Z
+    This class supports both formats.
```

Comment:
nit: "This class supports both formats as inputs and uses the hh:mmS as the output format."